index.js

```diff
@@ -284,6 +285,7 @@ class Hyperbee {
     this.sep = opts.sep || SEP
     this.readonly = !!opts.readonly
     this.prefix = opts.prefix || null
+    this.onlyIfChanged = opts.onlyIfChanged
```

Comment:
`!!opts.onlyIfChanged` ? (that way it's always a boolean, and the default is false)